src/docker/build/docker-image/Dockerfile

```
@@ -59,15 +59,11 @@ RUN sed -i 's/Components: main/Components: main non-free/' /etc/apt/sources.list
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)